### PR TITLE
Fix "unpredictable" sections

### LINKED ISSUE #140: Refactor Unpredictable vs. Extremely Unpredictable, add lessons
From @michael-scharf (https://mailarchive.ietf.org/arch/msg/mops/eGGPYW2vytPsLwRBlJ38MezPFh0/)

(In Section 3.6.  Unpredictable Usage Profiles / Section 3.7.  Extremely)

Unpredictable Usage Profiles

I am not fully convinced by the distinction between "unpredictable" and
"extremely unpredictable". To me, these two sections could be merged and maybe
also be shortened. More specifically, Section 3.7 lists a lot of statistics
from the past. What is IMHO a bit missing in Section 3.7 are actual operational
considerations. For instance, are there any lessons learnt for the future? Note
that Section 3.6 has a reasonable conclusion at the end.

### PR DESCRIPTION
**Pull request by SpencerDawkins**
close #140

### CODE REVIEW

File: draft-ietf-mops-streaming-opcons.md [hunk 1]
```diff
@@ -501,46 +479,33 @@ It is worth noting that not all high-demand content is "live" content. One relev
...
 
 ## Unpredictable Usage Profiles {#sec-unpredict}
 
-Although TCP/IP has been used with a number of widely used applications that have symmetric bandwidth requirements (similar bandwidth requirements in each direction between endpoints), many widely-used Internet applications operate in client-server roles, with asymmetric bandwidth requirements. A common example might be an HTTP GET operation, where a client sends a relatively small HTTP GET request for a resource to an HTTP server, and often receives a significantly larger response carrying the requested resource. When HTTP is commonly used to stream movie-length video, the ratio between response size and request size can become arbitrarily large.
+It is also possible for usage profiles to change significantly and suddenly. These changes are more difficult to plan for, but at a minimum, recognixing that sudden changes are happening is critical.
```
Comment (acbegen) [inline draft-ietf-mops-streaming-opcons.md:492]: s/recognixing/recognizing

File: draft-ietf-mops-streaming-opcons.md [hunk 2]
```diff
@@ -501,46 +479,33 @@ It is worth noting that not all high-demand content is "live" content. One relev
...
 
-The BitTorrent case study is just one example, but the example is included here to make it clear that unpredicted and unpredictable massive traffic spikes may not be the result of natural disasters, but they can still have significant impacts.
+* Many access networks for end users used underlying technologies that are inherently asymmetric, favoring downstream bandwidth (e.g. ADSL, cellular technologies, most IEEE 802.11 variants), assuming that most users will need more downstream bandwidth than upstream bandwidth. This is a good assumption for client-server applications such as streaming video or software downloads, but BitTorrent rewarded peers that uploaded as much as they downloaded, so BitTorrent users had much more symmetric usage profiles which interacted badly with these assymetric access network technologies.
```
Comment (acbegen) [inline draft-ietf-mops-streaming-opcons.md:500]: s/assymetric/asymmetric

File: draft-ietf-mops-streaming-opcons.md [hunk 3]
```diff
@@ -501,46 +479,33 @@ It is worth noting that not all high-demand content is "live" content. One relev
...
-- A 30% increase in upstream traffic over their pre-pandemic levels, and
-- A steady increase in the overall volume of DDoS traffic, with amounts exceeding the pre-pandemic levels by 40%. (He attributed this increase to the significant rise in gaming-related DDoS attacks ({{LabovitzDDoS}}), as gaming usage also increased.)
+### Impact of Global Pandemic {{IABcovid}}
 
-Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. Given a larger number of reports and more time to reflect, the following observations from the draft workshop report are worth considering.
+Early in 2020, the CoViD-19 pandemic and resulting quarentines and shutdowns led to significant changes in traffic patterns, resulting from a causes such as large number of people who were suddenly working and attending school remotely, and using more interactive applications (video conderencing, in addition to streaming media). Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. The following observations from the workshop report are worth considering.
```
Comment (acbegen) [inline draft-ietf-mops-streaming-opcons.md:508]: s/Early in 2020, the CoViD-19 pandemic and resulting quarentines and shutdowns led to significant changes in traffic patterns, resulting from a causes such as large number of people who were suddenly working and attending school remotely, and using more interactive applications (video conderencing, in addition to streaming media). Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. The following observations from the workshop report are worth considering.
/Early in 2020, the CoViD-19 pandemic and resulting quarantines and shutdowns led to significant changes in traffic patterns, due to a large number of people who suddenly started working and attending school remotely and using more interactive applications (video conferencing, in addition to streaming media). Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. The following observations from the workshop report are worth considering.

**Comment by acbegen** [comment]
Fixed a few typos, otherwise ready to merge.

File: draft-ietf-mops-streaming-opcons.md (hunk 1 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:492]: Got it.

File: draft-ietf-mops-streaming-opcons.md (hunk 2 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:500]: Got it.

**Comment by SpencerDawkins** [comment]
Merging on Ali's review.

**Pull request merged**